app/grandchallenge/evaluation/tasks.py

```diff
@@ -537,3 +548,21 @@ def assign_submission_permissions(*, phase_pk: uuid.UUID):
     )
     for sub in Submission.objects.filter(phase__id=phase_pk):
         sub.assign_permissions()
+
+
+@acks_late_micro_short_task
+def cancel_external_evaluations_past_timeout():
+    Evaluation = apps.get_model(  # noqa: N806
+        app_label="evaluation", model_name="Evaluation"
+    )
+    timeout_threshold = datetime.now() - timedelta(
+        seconds=settings.EXTERNAL_EVALUATION_TIMEOUT_IN_SECONDS
+    )
+
+    Evaluation.objects.filter(
```

Comment:
De we need to notify the followers of the evaluation?

Reply:
Currently, evaluations don't have followers, it's the phases that have followers (namely the challenge admins and the submission creators). For this case, I think we would only want to notify the external evaluator that claimed the evaluation, right?